### PR TITLE
BadgeBLE: add description of uploading behavior

### PR DESCRIPTION
**Pull request by kienvo**
Add detail on how data is sent/received through 16 bytes value of the characteristic

### CODE REVIEW

**Comment by simon-budig** [comment]
Hi @kienvo 

Thanks for your contribution. May I ask for a small rework?

Please keep the "format" column in the table. This is vital information for people trying to implement this.
One example is the "sizes" row, where there is an 8 element array of 16-bit values (so it is in fact 16 byte sized, that is an error in your description, also offset needs adjustment then). Offset and byte-size are useful informational additions, but they don't carry the same information as the type description in the "format" column.

Also for ease of editing I'd suggest to align the cell separators in the markdown with the separators in the header row.

The additional paragraph about splitting up that data block is good. I'd probably add a small sentence for the padding of the last 16-byte block.

Thanks!

**Comment by kienvo** [comment]
@simon-budig Thank you for the review. I have adjusted to your suggestions. Please take a look at my new changes.

File: BadgeBLE.md
Comment (AnyTimeTraveler) [inline BadgeBLE.md:1]: What you could do for the padding at the end, is write it out as a formula:
`10h - ((40h + 11*N) mod 10h)`

That should give the correct number of padding bytes. If you leave a questionmark there, it looks like it hasn't been discovered yet.

Comment (simon-budig) [reply BadgeBLE.md:1]: unfortunately that formula is quite complicated and might add to the confusion. I'd probably prefer specifying a range like 0-15 bytes. *shrug*

Anyway. The rest of the changes looks good, I'll accept it.

Comment (AnyTimeTraveler) [reply BadgeBLE.md:1]: I'm very much in favor of the byte range.
Together with the comment below, it should explain it very well.

**Pull request merged**